### PR TITLE
main/usbredir: upgrade to 0.8.0

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

File: main/usbredir/APKBUILD [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
 pkgname=usbredir
-pkgver=0.7.1
+pkgver=0.8.0
 pkgrel=0
 pkgdesc="USB network redirection protocol libraries"
 url="http://spice-space.org/page/UsbRedir"
 arch="all"
 options="!check"  # No test suite.
 license="GPL-2.0-or-later LGPL-2.1-or-later"
-depends_dev="libusb-dev"
```
Comment (tcely) [inline main/usbredir/APKBUILD]: Why is `libusb-dev` being removed from `depends_dev` here?

Comment (maxice8) [reply main/usbredir/APKBUILD]: dependency tracing already finds pc:libusb-1.0 so it is redundant

Comment (tcely) [reply main/usbredir/APKBUILD]: It's not unless `libusb-dev` is the package that provides `pc:libusb-1.0`.

Comment (maxice8) [reply main/usbredir/APKBUILD]: image

Comment (tcely) [reply main/usbredir/APKBUILD]: I stand corrected.

```
# apk info --provides libusb-dev
libusb-dev-1.0.22-r0 provides:
pc:libusb-1.0=1.0.22
```

File: main/usbredir/APKBUILD [hunk 2]
```diff
@@ -1,17 +1,15 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=usbredir
```
Comment (tcely) [inline main/usbredir/APKBUILD]:
```suggestion
_distname="usbredir"
pkgname="${_distname}"
```

File: main/usbredir/APKBUILD [hunk 3]
```diff
@@ -1,17 +1,15 @@
...
 arch="all"
 options="!check"  # No test suite.
 license="GPL-2.0-or-later LGPL-2.1-or-later"
-depends_dev="libusb-dev"
-makedepends="$depends_dev"
+makedepends="libusb-dev"
 subpackages="$pkgname-dev $pkgname-doc $pkgname-server"
-source="https://www.spice-space.org/download/usbredir/usbredir-$pkgver.tar.bz2
-	"
+source="https://www.spice-space.org/download/usbredir/usbredir-$pkgver.tar.bz2"
```
Comment (tcely) [inline main/usbredir/APKBUILD]:
```suggestion
source="https://www.spice-space.org/download/${_distname}/${_distname}-${pkgver}.tar.bz2"
```

File: main/usbredir/APKBUILD [hunk 4]
```diff
@@ -1,17 +1,15 @@
...
 
 builddir="$srcdir"/usbredir-$pkgver
```
Comment (tcely) [inline main/usbredir/APKBUILD]:
```suggestion
builddir="${srcdir}/${_distname}-${pkgver}"
```

**Comment by maxice8** [comment]
looks over-engineered with no benefit. i'll just replace $pkgname where applicable.

**Comment by maxice8** [comment]
On Tue, May 07, 2019 at 12:55:56PM +0000, tcely wrote:
> > looks over-engineered with no benefit. i'll just replace $pkgname where applicable.
> 
> I agree with you but @Ikke has argued for making changing `pkgname` easy in the past.
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/7283#issuecomment-490067579

That just moves the problem around and makes it less readable.

**Comment by tcely** [comment]
It does as stated, unlinks pkgname (aports controlled) from the distribution name (upstream controlled) so that renaming the aport won't break the `source` or `builddir` paths.

**Pull request closed without merging**